archive-interviewees.php

```diff
@@ -4,9 +4,8 @@
  *
  * @link https://github.com/MITLibraries/music-oral-history
  *
- * @package WordPress
- * @subpackage music-oral-history
- * @since 1.1.11
+ * @package music_oral_history
+ * @since 2.0.0
```

Comment:
Is there a reason why this `since` tag is being updated? The way I understood that tag was that it identifies when a template is _first_ added - so it doesn't then get updated.

If I've misunderstood how to use the tag, then I apologize for the confusion.

Reply:
it seems you know better how to use the tag than me. I will update